partials/members/feedback-success.hbs

```diff
@@ -0,0 +1,9 @@
+<div
+    role="alert"
+    class="alert alert--success members-form__feedback members-form__feedback--success"
+>
```

Comment:
Tady žádný `data-members-success` nemusí být?

Reply:
Nemusí, podle všeho je to [jen pro chybové hlášky](https://ghost.org/docs/members/signup-signin-forms/#error-messages).